README.rst

```diff
@@ -18,45 +31,48 @@ or, if you know you want to create a development Account, you can omit that argu
 
 There are two constants available to you to specify the url for either the development environment
 or the production environment, which are:
-- `fxapom.DEV_URL` - the url for the development environment
-- `fxapom.PROD_URL` - the url for the production environment
+* `fxapom.DEV_URL` - the url for the development environment
+* `fxapom.PROD_URL` - the url for the production environment
 
 Example of creating an account on the development environment, using the default:
-```python
-from fxapom.fxapom import FxATestAccount
-acct = FxATestAccount().create_account()
-```
+
+.. code-block:: python
+
+  from fxapom.fxapom import FxATestAccount
+  acct = FxATestAccount().create_account()
+
 
 Example of creating an account on the development environment, specifying the `DEV_URL`:
-```python
-from fxapom.fxapom import DEV_URL, FxATestAccount
-acct = FxATestAccount(DEV_URL).create_account()
-```
+
+.. code-block:: python
+
+  from fxapom.fxapom import DEV_URL, FxATestAccount
+  acct = FxATestAccount(DEV_URL).create_account()
 
 To sign in via Firefox Accounts, use the `sign_in` method in the `WebDriverFxA` object,
 passing in the email addresss and password.
 
 Example:
```

Comment:
I think we can remove this, and just add `:` to the end of the previous sentance.